wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/generation/Generation.kt

```diff
@@ -221,6 +222,16 @@ private fun Metadata.linkedMapOfInputs(inputTypings: Map<String, Typing>): CodeB
     }
 }
 
+private fun TypeSpec.Builder.addMaybeDeprecated(coords: ActionCoords): TypeSpec.Builder {
+    val newerClass = coords.copy(version = coords.deprecatedByVersion ?: return this)
```

Comment:
I propose to make it clear when this function is a no-op (I know, it's longer, but I found it pretty confusing to find this `return this` inside the parenthesis. Shorter is not always more readable :sweat_smile:):
```suggestion
    if (coords.deprecatedByVersion == null) {
        return this
    }

    val newerClass = coords.copy(version = coords.deprecatedByVersion)
```